packages/gluestick/src/config/__tests__/compileWebpackConfig.test.js

```diff
@@ -14,7 +19,18 @@ const loggerMock = {
   error: jest.fn(),
 };
 
+const originalProcessCwd = process.cwd.bind(process);
```

Comment:
To clone it using `bind`, I follow this advice https://github.com/getify/You-Dont-Know-JS/blob/master/this%20%26%20object%20prototypes/ch2.md#safer-this.